docs/node/task-queues.md

```diff
@@ -37,20 +37,11 @@ const worker = await Worker.create({
 });
 ```
 
-3. Optionally, when calling an Activity, you can specify the task queue by passing the `taskQueue` option to `Context.configureActivities()`. If you do not specify a `taskQueue`, then the Node SDK places Activity Tasks in the same Task Queue as the Workflow Task Queue.
+3. Optionally, in Workflow code, when calling an Activity, you can specify the task queue by passing the `taskQueue` option to [`configureActivities()`](https://nodejs.temporal.io/api/namespaces/workflow#configureactivities). If you do not specify a `taskQueue`, then the Node SDK places Activity Tasks in the same Task Queue as the Workflow Task Queue.
 
 ```ts
-import { Context } from '@temporalio/workflow';
-import * as activities from '../activities';
-
-const { greet } = Context.configureActivities<typeof activities>({
+const { greet } = configureActivities<typeof activities>({
   taskQueue: 'my-other-task-queue',
   startToCloseTimeout: '1s',
 });
```

Comment:
this api bothers me a bit. `greet` appears out of nowhere, guaranteed only by `typeof activities`.

what do we think about something that more intentionally mirrors the workflow stubbing:

```tsx
const greet = new Activity({
  type: activities.greet,
  taskQueue: 'my-other-task-queue',
  startToCloseTimeout: '1s'
});
```

Reply:
The won't work because `activities.greet` is just the type and cannot be used as an argument.
This is the best way I find to get a typed and named reference to an activity without actually importing the activities into the Workflow isolate (which is not supported).

Reply:
this is a good answer - we should actually explain how activities are injected into the WF isolate, as I'm sure this will come up again. giving users a good mental model will save us 100 questions down the line